screen/features/build.gradle.kts

```diff
@@ -12,4 +12,6 @@ dependencies {
     implementation(projects.shared.ui.core)
     implementation(projects.shared.ui.core)
     implementation(projects.shared.ui.navigation)
+    implementation(projects.temp.legacyCode)
```

Comment:
Let's not add temp and legacy code. I understand the UI is better but in that case re-implement by styling m3 components. I want to get rid of this legacy and inefficient code because it brings complexity. 

That's why it's in `temp`

Reply:
Done